go.mod

```
@@ -0,0 +1,18 @@
+module nylas-cli
```

```suggestion
module github.com/nylas/cli
```

Since we intend to opensource this.